DEPLOYING.md

```diff
@@ -69,52 +69,44 @@ contains step-by-step deployment instructions using opeNode's command line inter
 This same process is outlined in the next section.
 
 #### Deploy your app to your new instance
-1. On your local machine, open a command prompt.
+Note: A video walkthrough can be found here to supplment the instructions below: https://www.youtube.com/watch?v=SRzQOEwR4dM&list=PLmFHH6TuGBX1GjMAfzwz1wYz274RHq4ml&index=7&t=0s 
 
+1. On your local machine, open a command prompt.
+ 
 2. If needed, install the opeNode command line interface (only needs to be done once - this requires
 the command prompt to be launched in administrator mode):
 ```
 npm install -g openode
 ```
-
+ 
 3. Change directory to your project:
 ```
 cd YOUR_PROJECT_DIRECTORY
 ```
-
-4. If needed, create a Dockerfile using the openode CLI (only needs to be done once per project):
+4. Use the Dockerfile that comes with any of the sample MRE projects. A second option is you can create a Dockerfile using the openode CLI (only needs to be done once per project):
 ```
 openode template
 ```
-This creates a file on disk named `Dockerfile`
-
+This creates a file on disk named `Dockerfile`. Make sure you change where it says  WORKDIR /opt/app to WORKDIR /opt/mre or it will not be able to find /opt/app as a module. 
+ 
 5. Edit the Dockerfile
 The Node version specified in the Dockerfile is newer than the MRE SDK's Node depencency. To eliminate
 potential incompatibility, edit the first line of the Dockerfile so that it specifies Node v8.12:
 ```
 FROM node:8.12.0-alpine
 ```
```

Comment:
Could you remove the instructions about setting the node version to 8.12? Modern Node.js runs MREs just fine.